src/peer_connection/certificate.rs

```diff
@@ -71,7 +72,10 @@ impl RTCCertificate {
         };
         params.key_pair = Some(key_pair);
 
-        let expires = params.not_after.into();
+        //let expires = params.not_after.into();
+        // Workaround for issue overflow when adding duration to instant on armv7
+        // https://github.com/webrtc-rs/examples/issues/5 https://github.com/chronotope/chrono/issues/343
+        let expires = SystemTime::now().add(Duration::from_secs(172800)); //60*60*48 or 2 days
```

Comment:
let’s use conditional compiler https://doc.rust-lang.org/reference/conditional-compilation.html  to handle arm special case, other arch still use line 75